src/js/breadcrumbs/specs/breadcrumbs-spec.js

```diff
@@ -0,0 +1,44 @@
+(function (define) {
+    'use strict';
+
+    define(['jquery',
+            'underscore',
+            '../breadcrumbs-view.js',
+            '../breadcrumbs-model.js'
+           ],
+           function($, _, BreadcrumbsView, BreadcrumbsModel) {
+
+               describe('Breadcrumbs component', function () {
+                   var model, view;
+
+                   var testBreadcrumbs = function (breadcrumbs) {
+                       model.set('breadcrumbs', breadcrumbs);
+                       expect(view.$('nav.breadcrumbs').length).toBe(1);
+                       _.each(view.$('.nav-item'), function (el, index) {
+                           expect($(el).attr('href')).toEqual(breadcrumbs[index].url);
+                           expect($(el).text()).toEqual(breadcrumbs[index].title);
+                       });
+                   };
+
+                   beforeEach(function () {
+                       model = new BreadcrumbsModel();
+                       view = new BreadcrumbsView({
+                           model: model
+                       });
+                   });
+
+                   it('does not show breadcrumbs by default', function () {
+                       expect(view.$el.html()).not.toContain('<nav class="breadcrumbs">');
+                   });
+
+                   it('shows breadcrumbs if they are supplied', function () {
+                       testBreadcrumbs([
```

Comment:
Nit: I like to have the except calls explicitly in the body of the test so that it is easy to read what is being tested. If you use ``SpecHelpers.withData`` then you won't need the ``testBreadcrumbs`` function which will naturally make this happen.